lib/CAPI/ExportFIRRTL/ExportFIRRTL.cpp

```diff
@@ -17,8 +17,8 @@ using namespace circt;
 using namespace firrtl;
 
 MlirLogicalResult mlirExportFIRRTL(MlirModule module,
-                                   MlirStringCallback callback, void *userData,
-                                   FIRVersion version) {
+                                   MlirStringCallback callback,
+                                   void *userData) {
   mlir::detail::CallbackOstream stream(callback, userData);
   return wrap(exportFIRFile(unwrap(module), stream, {}, {3, 0, 0}));
```

Comment:
So this always exports with FIRVersion 3.0.0? What was the intent for having the FIRVersion argument to mlirExportFIRRTL @seldridge? Should it just be removed from this C API and hardcoded to 3.0.0, like this PR does?

Reply:
I think the setting `FIRVersion` to latest version for a specific version of CIRCT is OK? Since not like parsing which may parse out-of-date `fir`, emit is just emit `fir` under THIS CIRCT version.

If `FIRVersion` in `mlirExportFIRRTL` is exported to C-API, I don't see any reason for Chisel user to select the FIRRTL version they choose to emit, since this is also harden to latest version in Chisel.

IMO, I think CIRCT should add `FIRVersion::latested` method a possible solution to avoid we forget to change this in the future.

Reply:
@seldridge So may I merge this PR at the moment?

Reply:
Ah sorry, I just misunderstood your comment, I have added a static function in the new commit.